universal-application-tool-0.0.1/app/services/cloud/FileNameFormatter.java

```diff
@@ -0,0 +1,31 @@
+package services.cloud;
+
+import views.applicant.ApplicantProgramBlockEditView;
+
+public class FileNameFormatter {
```

Comment:
suggestion (blocking): please document this class. The filename prefix convention is confusing and not well documented so this would be an excellent place to do it.